gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -204,7 +204,7 @@ static void waitJobCompletion(HelixManager helixManager, String workFlowName, St
       } else {
         // We have waited for WorkflowContext to get initialized,
         // so it is found null here, it must have been deleted in job cancellation process.
-        log.info("WorkflowContext not found. Job is probably cancelled.");
+        log.info("WorkflowContext not found. Job {} is probably cancelled.", jobName);
```

Comment:
Seems unrelated to the change proposed in this PR. Can we remove this?

Reply:
I added this in the description. Do you want me to create a separate PR for this?

Reply:
Even though it is a trivial one-line change, I prefer to keep unrelated changes in separate PRs. The context of such unrelated changes is lost after some time and can be confusing to a new developer. My suggestion is to pull this change out of this PR.